plugins/reach/lib/kReachManager.php

```diff
@@ -54,6 +212,13 @@ public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedCol
 		if ($object instanceof categoryEntry && $object->getStatus() == CategoryEntryStatus::ACTIVE)
 			return true;
 
+		if ($object instanceof entry && $object->getType() == entryType::MEDIA_CLIP && $object->isModified() )
+		{
+			$event = new kObjectChangedEvent($object,$modifiedColumns);
```

Comment:
If there are new rules, must act only according to the profile rules decision, else to support backward, check the legacy code.,